libraries/AP_HAL_ChibiOS/hwdef/MambaF405/hwdef.dat

```diff
@@ -0,0 +1,145 @@
+# hw definition file for processing by chibios_pins.py
+# Mamba F405 MK2
+# with F405 mcu, mpu6000 imu and 7456 series osd
+
+# MCU class and specific type.
+MCU STM32F4xx STM32F405xx
+
+HAL_CHIBIOS_ARCH_F405 1
+
+# board ID for firmware load
+APJ_BOARD_ID 1019
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+STM32_PLLM_VALUE 8
+
+# board voltage
+STM32_VDD 330U
+
+STM32_ST_USE_TIMER 5
+
+# order of I2C buses
+I2C_ORDER I2C1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART6 USART3 USART1
+
+# The pins that USB is connected on
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+PC5 VBUS INPUT OPENDRAIN
+
+# USART1 is wired to SBUS input
+PA10 TIM1_CH3 TIM1 RCININT FLOAT LOW
```

Comment:
Do you want to provide an alt config so that you can use this as a regular uart? Look at the KakauteF7Mini hwdef fro an example

Reply:
Nice feat, I'll do that !